data/happenings.xml

```diff
@@ -1 +1,13710 @@
-happenings-20131125:26.xml
\ No newline at end of file
+<WORLD><HAPPENINGS><EVENT id="7975081">
```

Comment:
You appear to have changed the symlink to point to the contents of the file, instead of to the file. This doesn't work for me when I check it out.

Do symlinks checked out by git just not work right on windows?